src/commands/secrets/download.ts

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs-extra';
+import yaml from 'js-yaml';
+import path from 'path';
+import tmp from 'tmp';
+import untildify from 'untildify';
+import AccountUtils from '../../architect/account/account.utils';
+import { EnvironmentUtils } from '../../architect/environment/environment.utils';
+import Command from '../../base-command';
+import { SecretsDict } from '../../dependency-manager/secrets/type';
+
+tmp.setGracefulCleanup();
+
+export default class SecretsDownload extends Command {
+  async auth_required(): Promise<boolean> {
+    return false;
```

Comment:
This should be true. If a user is not logged in there are no accounts to get secrets from.

Reply:
Updated to `true`